core/system/src/mpu/vmpu_freescale_k64_mem.c

```diff
@@ -63,21 +63,27 @@ void vmpu_mem_switch(uint8_t src_box, uint8_t dst_box)
 
 void vmpu_mem_init(void)
 {
-    /* enable read access to unsecure flash regions - allow execution */
+    /* enable read access to unsecure flash regions
+     * - allow execution
+     * - give read access to ENET DMA bus master */
     vmpu_mpu_set_static_acl(1, FLASH_ORIGIN, (uint32_t) __uvisor_config.secure_end - FLASH_ORIGIN,
         UVISOR_TACL_SREAD |
         UVISOR_TACL_SEXECUTE |
         UVISOR_TACL_UREAD |
         UVISOR_TACL_UEXECUTE |
-        UVISOR_TACL_USER);
+        UVISOR_TACL_USER,
+        0x4UL<<18);
```

Comment:
Could you add spaces in `0x4UL << 18)`? (also in the other occurrence below)

BTW, do we want to provide an interface/macro for selecting a certain bus master or to set the K64F-specific ACLs? The hardcoded numbers are not very user-friendly. If we don't want to do it now we can add and issue/story.